package.json

```diff
@@ -75,6 +75,6 @@
       "/node_modules/",
       "/tmp/"
     ],
-    "testRunner": "jest-jasmine2"
+    "testRunner": "jest-circus"
```

Comment:
We can remove this line now, as the default is `jest-circus`.

Reply:
Removed as requested.